test/common/index.js

```diff
@@ -24,6 +24,8 @@
 const process = global.process;  // Some tests tamper with the process global.
 const path = require('path');
 const fs = require('fs');
+/* eslint-disable no-restricted-properties */
+const { openSync } = require('fs');
```

Comment:
...since we're already importing all of `fs` on the immediately prior line, is this really necessary? Can we just use `fs.openSync` in the one place it gets used later on (line 814)?